tests/src/main/python/TestDriver_MeasureCLI.py

```diff
@@ -0,0 +1,64 @@
+
+import unittest
+import os
+import time
+import csv
+import json
+import re
+import subprocess
+
+currentDir=os.getcwd()
+baseDir = currentDir + '/'
+testFile=baseDir + os.environ['TESTS_JSON']
+jar = os.environ['JAR']
+
+def setup():
+    os.chdir(baseDir)
+    tests = list()
+    with open(testFile) as f:
+        data = json.load(f)
+        testValues = data['tests']
+        for testValue in testValues.values():
+            tests.append((testValue['params'], testValue['targets'], testValue['response'], testValue['resource'], testValue['measureServer']))
+    return tests
+ 
+class Test(object):
+
+    @pytest.mark.parametrize("params, targets, output, resource, measureServer",setup())
+    def test1(self, params, targets, output, resource, measureServer):
+        self.execute(params, targets, output, resource, measureServer)
+
+    # Execute submits a query and validates the return.
+    def execute(self, params, targets, output, resource, measureServer):
+        output = re.sub('@\w+', '@',output)
+        o = output.split('\n')
+        callDetails = ["java", "Xms1G", "-Djavax.net.ssl.trustStore=config/trustStore.pkcs12", "-Djavax.net.ssl.trustStorePassword=change-password", "-Djavax.net.ssl.trustStoreType=pkcs12", "-jar", jar]
```

Comment:
Hard-coding the details to the truststore, including the password, is generally not a good idea. You should probably pass those in from the environment like you are the other options.

Reply:
Thank you!

Reply:
Checked in changes. Instead of in the yaml, which also get checked in. We use an environment file that already contains information that should never be checked in. Decided to add these details there.